lib/commands/createOrUpdatePassword.ts

```diff
@@ -1,8 +1,8 @@
 import type { FetchResult } from "apollo-link";
 import gql from "graphql-tag";
 
-import { CreateOrUpdatePasswordInput, User } from "@lib/@types/ProviderUser";
-import { fetchManagement } from "@src/utils/fetchManagement";
+import { fetchManagement } from "../../src/utils/fetchManagement";
+import { CreateOrUpdatePasswordInput, User } from "../@types/ProviderUser";
```

Comment:
Is this because of path aliases not being processed by `ts-node`?

Reply:
Yes, we need to change path aliases to relative ones every time a file is used in `ts-node` environment

Reply:
Can you make a story to add path aliasing to `ts-node` config please? :D

Reply:
You mean in ClickUp ?

Reply:
Damn, `Clubhouse` throwback haha. Yeah, I meant _task_.